apps/docs/src/components/document/components/component-status-table.tsx

```diff
@@ -52,8 +52,11 @@ export function ComponentStatusBlock({ componentId }: { componentId: string }) {
       const figmaPlatform = component.statuses.find(
         (platformStatus) => platformStatus.platform.id === 'figma',
       );
-      const htmlPlatform = component.statuses.find(
-        (platformStatus) => platformStatus.platform.id === 'html',
+      const localPlatform = component.statuses.find(
+        (platformStatus) => platformStatus.platform.id === 'local',
```

Comment:
suggestion (non-blocking): Usage of constants for `local`, `html`, `figma` and `react`.

Reply:
it's a type `export type ComponentPlatformId = 'figma' | 'local' | 'global' | 'react';`